docs/guides/node-operator-manual.md

```diff
@@ -276,3 +326,35 @@ docker run --rm -it \
   --datadir /root/.lighthouse/data \
   --directory /root/validator_keys
 ```
+
+### Checking the keys of all Lido Node Operators
+
+After keys are submitted, it's encouraged to wait for the transactions to be mined and to check how they appear on the chain.
```

Comment:
"Please, check the submitted keys before starting the vote on increasing the key limit."

Reply:
I don't think this is explicit enough. I suggest this:
```
### Checking the keys of all Lido Node Operators

After keys are submitted, it's critical to first wait for the key submission transactions to be mined and to verify them with all Lido keys after they appear on the blockchain.

Never vote for increasing the key limits of Node Operators before verifying new keys are present and valid.
```